### DISCUSSION TITLE
Steam Streaming Microphone through Sunshine? Is it possible?

### DISCUSSION THREAD

**Discussion by mxp83** [LizardByte/community#336] [Sunshine]
If I set my Host pc mic to the Streaming Mic and use my Steam Deck built in mic through Moonlight, will the host pc receive the audio?

**Comment by ReenigneArcher** [comment]
There is no microphone support currently.

**Reply by febs** [reply]
Any chances for this to be implemented? It would be a godsend.

Thanks,

**Reply by tmplt** [reply]
Is there a tracking issue for this feature?

**Reply by ReenigneArcher** [reply]
We don't use issues for feature requests.

**Reply by charlysotelo** [reply]
What do you use (if anything) to track upcoming features?

**Reply by ReenigneArcher** [reply]
https://github.com/LizardByte/Sunshine/pulls

**Reply by ReenigneArcher** [reply]
feature request: https://github.com/orgs/LizardByte/discussions/129
